server/config/brain/chat/questions.en.json

```diff
@@ -1,12 +1,63 @@
 [
   {
     "label": "chat.greeting",
-    "questions": ["Hey!"],
-    "answers": ["Hey!"]
+    "questions": ["Hey!", "Hi", "Hello"],
+    "answers": ["Hey!", "Hi", "Hello, I'm Gladys."]
+  },
+  {
+    "label": "chat.greeting-gladys",
+    "questions": ["Hey Gladys !", "Hi Gladys", "Hello Gladys"],
+    "answers": [
+      "Hey, I'm your home automation assistant !",
+      "Hello, I am your home automation assistant !",
+      "Hi, I am your home automation assistant !"
+    ]
   },
   {
     "label": "chat.i-love-you",
     "questions": ["I love you"],
-    "answers": ["Me too!", "Of course you love me..."]
+    "answers": ["Me too!", "I knew that ..."]
+  },
+  {
+    "label": "chat.joke",
+    "questions": ["Gladys tell me a joke.", "Gladys do you know a joke ?", "Tell me a joke."],
+    "answers": [
+      "Do you know the devil's joke? Too bad it's hell !",
+      "The other day I told a joke about my clothes. They were folded !",
```

Comment:
Not sure if the English works on this one, or I don't understand the joke. All the others are great though.

Reply:
Je vais m'exprimer en Français, car je ne suis pas 100% à l'aise en anglais.
C'est un jeu de mot, ça fonctionne en français mais aussi en anglais. Vous n'allez pas être "plié de rire", ça c'est sure.
La plupart des blagues dans Google vous font rire? Il y a beaucoup de jeux de mot.
Je peux vous laisser la main afin de rajouter autant de blagues à mourir de rire que vous souhaitez.
C'est un premier ajout, qui sera enrichi en fonction des retours.

Reply:
@tlse-vins Brad explique que la blague ne fonctionne pas en anglais  car "être plié de rire" est une expression française qui n'existe pas en Anglais. Tu devrai l'enlever ;) , je penses qu'il à raison.

@thebradleysanders thank you for feedback btw.